packages/create-nx-plugin/package.json

```diff
@@ -30,7 +30,7 @@
   "homepage": "https://nx.dev",
   "dependencies": {
     "create-nx-workspace": "file:../create-nx-workspace",
-    "chalk": "^4.1.0",
+    "picocolors": "^1.1.0",
```

Comment:
The repo doesn't use workspaces, so please add this to the root package.json too

Reply:
This is at least in part why your CI run failed

Reply:
Ah I somehow totally missed that. Had been wondering what kind of setup it was meant to be

Thanks for the explanation